src/NuGet.Core/NuGet.Commands/MSBuildPackTargetArgs.cs

```diff
@@ -15,6 +15,7 @@ public class MSBuildPackTargetArgs
         public HashSet<string> AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder { get; set; }
         public string AssemblyName { get; set; }
         public string NuspecOutputPath { get; set; }
+        public string PackageOutputPath { get; set; }
```

Comment:
Why do we need this? The output path is already provided by other means.

Reply:
What other means and Where?

Reply:
Redirect the packageoutputpath works in some capacity today right? 

https://github.com/NuGet/NuGet.Client/blob/1fcb0dc2507330453e41bc54a917e20356a85055/src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTask.cs#L42